### PR TITLE
DHFPROD-628 Adding quickstart tests for PII property

### PR DESCRIPTION
**Pull request by rahulvudutala**
Creating a PR to test the PII feature for the quick-start application

Added the following tests:
1) Enable PII property from quickstart UI
2) Toggle PII on a property
3) Check for the persistence of the PII property when logged in a different session
 
Also, added/updated the locators for the entities

### CODE REVIEW

File: quick-start/e2e/specs/create/create.ts [hunk 1]
```diff
@@ -168,7 +170,28 @@ export default function() {
       browser.wait(EC.presenceOf(entityPage.toast));
       browser.wait(EC.stalenessOf(entityPage.toast));
     });
-
+    
+    it ('should toggle pii button for product entity', function() {
+      console.log('ad');
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: what is "ad"?

File: quick-start/e2e/specs/create/create.ts [hunk 2]
```diff
@@ -168,7 +170,28 @@ export default function() {
...
+      browser.executeScript('window.document.getElementsByClassName("edit-start")[1].click()');
+      browser.wait(EC.visibilityOf(entityPage.entityEditor));
+      expect(entityPage.entityEditor.isPresent()).toBe(true);
+      let skuProperty = entityPage.getPropertyByPosition(1);
+      expect(entityPage.hasClass(entityPage.getPropertyPii(skuProperty), 'active')).toBe(true);
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: why there are multiple verification with conflicting values? One is true and one is false? Shouldn't it only be true?

File: quick-start/e2e/specs/create/create.ts [hunk 3]
```diff
@@ -168,7 +170,28 @@ export default function() {
...
+      entityPage.getPropertyPiiColumn(skuProperty).click();
+      expect(entityPage.hasClass(entityPage.getPropertyPii(skuProperty), 'active')).toBe(false);
+      entityPage.getPropertyPiiColumn(skuProperty).click();
+      expect(entityPage.hasClass(entityPage.getPropertyPii(skuProperty), 'active')).toBe(true);
+      skuProperty = entityPage.getPropertyByPosition(2);
+      expect(entityPage.hasClass(entityPage.getPropertyPii(skuProperty), 'active')).toBe(false);
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: why there is 2nd sku?

File: quick-start/e2e/specs/create/create.ts [hunk 4]
```diff
@@ -190,7 +213,7 @@ export default function() {
       lastProperty = entityPage.lastProperty;
       entityPage.getPropertyName(lastProperty).sendKeys('price');
       entityPage.getPropertyType(lastProperty).element(by.cssContainingText('option', 'decimal')).click();
-      entityPage.getPropertyDescription(lastProperty).sendKeys('price description');
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: why do we need to change it to "pricedesc"? The verification later is still "price description"

File: quick-start/e2e/specs/create/create.ts [hunk 5]
```diff
@@ -214,54 +238,67 @@ export default function() {
...
+        let skuProperty = entityPage.getPropertyByPosition(1);
+        expect(entityPage.getPropertyName(skuProperty).getAttribute('value')).toEqual('sku');
+        expect(entityPage.getPropertyType(skuProperty).getAttribute('ng-reflect-model')).toEqual('string');
+        expect(entityPage.getPropertyDescription(skuProperty).getAttribute('value')).toEqual('sku description');
+        expect(entityPage.hasClass(entityPage.getPropertyPrimaryKey(skuProperty), 'active')).toBe(true);
+        expect(entityPage.hasClass(entityPage.getPropertyPii(skuProperty), 'active')).toBe(true);
+        let priceProperty = entityPage.getPropertyByPosition(2);
+        expect(entityPage.getPropertyName(priceProperty).getAttribute('value')).toEqual('price');
+        expect(entityPage.getPropertyType(priceProperty).getAttribute('ng-reflect-model')).toEqual('decimal');
+        expect(entityPage.getPropertyDescription(priceProperty).getAttribute('value')).toEqual('price description');
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: On the above code, you changed the description to "pricedesc"

File: quick-start/e2e/specs/create/create.ts [hunk 6]
```diff
@@ -214,54 +238,67 @@ export default function() {
...
-      expect(entityPage.hasClass(entityPage.getPropertyPrimaryKey(skuProperty), 'active')).toBe(true);
-      let priceProperty = entityPage.getPropertyByPosition(2);
-      expect(entityPage.getPropertyName(priceProperty).getAttribute('value')).toEqual('price');
-      expect(entityPage.getPropertyType(priceProperty).getAttribute('ng-reflect-model')).toEqual('decimal');
-      expect(entityPage.getPropertyDescription(priceProperty).getAttribute('value')).toEqual('price description');
-      expect(entityPage.hasClass(entityPage.getPropertyRangeIndex(priceProperty), 'active')).toBe(true);
-      entityPage.cancelEntity.click();
-      browser.wait(EC.invisibilityOf(entityPage.entityEditor));
-    });
+    		entityPage.isLoaded();
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: please make indentation the same

File: quick-start/e2e/specs/create/create.ts [hunk 7]
```diff
@@ -214,54 +238,67 @@ export default function() {
...
-      expect(entityPage.getPropertyDescription(productsProperty).getAttribute('value')).toEqual('products description');
-      expect(entityPage.hasClass(entityPage.getPropertyWordLexicon(productsProperty), 'active')).toBe(true);
-      // verify duplicate property is not created, the count should be 3
-      entityPage.getPropertiesCount().then(function(props){expect(props === 3)});
-      entityPage.cancelEntity.click();
-      browser.wait(EC.invisibilityOf(entityPage.entityEditor));
-    });
-
+        console.log('verify properties to Order entity');
+		entityPage.isLoaded();
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: please make indentation the same

File: quick-start/e2e/specs/create/create.ts [hunk 8]
```diff
@@ -214,54 +238,67 @@ export default function() {
       browser.wait(EC.stalenessOf(entityPage.toast));
     });
 
+    it ('should logout and login', function() {
+    		entityPage.logout();
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: please make indentation the same

File: quick-start/e2e/specs/create/create.ts [hunk 9]
```diff
@@ -214,16 +243,28 @@ export default function() {
       browser.wait(EC.stalenessOf(entityPage.toast));
     });
 
+    it ('should logout and login', function() {
+    	  entityPage.logout();
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: one extra space for indentation, please make it the same

File: quick-start/e2e/specs/create/create.ts [hunk 10]
```diff
@@ -214,16 +243,28 @@ export default function() {
...
+      loginPage.isLoaded();
+      loginPage.clickNext('ProjectDirTab');
+      browser.wait(EC.elementToBeClickable(loginPage.environmentTab));
+      loginPage.clickNext('EnvironmentTab');
+      browser.wait(EC.elementToBeClickable(loginPage.loginTab));
+      loginPage.login();
+    });
+    
     it ('should verify properties to Product entity', function() {
+    	  entityPage.isLoaded();
```
Comment (ayuwono) [inline quick-start/e2e/specs/create/create.ts]: one extra space for indentation, please make it the same

**Comment by ayuwono** [comment]
@grechaw, I've approved this. If you can review and approve it, we can merge this PR

**Comment by grechaw** [comment]
OK @ayuwono will do, Thanks!

**Pull request merged**